### PR TITLE
Accessibility translation

### PR DESCRIPTION
**Pull request by ElRodrigote**

### CODE REVIEW

**Comment by alejandronanez** [comment]
Hey @ElRodrigote, what's the state of this PR? Are you still working on it?

Thanks!

**Comment by ElRodrigote** [comment]
@alejandronanez  Hi!

Yes, I had a few personal issues this week but I'm working on it, I'll update now what I have so far. Sorry for the delay and not giving you guys an update status.

**Comment by alejandronanez** [comment]
all good @elrodrigote no problem! thanks for letting us know!

**Comment by ElRodrigote** [comment]
Hi @alejandronanez !

Finally I've finished translating and proof reading the file, it was quite extensive haha, sorry for the delay, If you could please take a look and review it, it would be awesome. 

I have a quick question, regarding titles having some kind of reference between braces. I've deleted them in the file, but would like to know if I should:
· Keep them as they are
· Translate them in situ
· Erase them

Cheers!

File: content/docs/accessibility.md [hunk 1]
```diff
@@ -1,32 +1,32 @@
...
-title: Accessibility
+title: Accesibilidad
 permalink: docs/accessibility.html
 ---
 
-## Why Accessibility? {#why-accessibility}
+## ¿Por qué Accesibilidad?
 
-Web accessibility (also referred to as [**a11y**](https://en.wiktionary.org/wiki/a11y)) is the design and creation of websites that can be used by everyone. Accessibility support is necessary to allow assistive technology to interpret web pages.
+La accesibilidad web (también conocida como [** a11y **] (https://en.wiktionary.org/wiki/a11y)) es el diseño y la creación de sitios web que pueden ser utilizados por todos. El soporte de accesibilidad es necesario para permitir que la tecnología de asistencia interprete las páginas web.
```
Comment (alejandronanez) [inline content/docs/accessibility.md]: Remove those spaces, all the document is in bold now 😅 
```suggestion
La accesibilidad web (también conocida como [**a11y**] (https://en.wiktionary.org/wiki/a11y)) es el diseño y la creación de sitios web que pueden ser utilizados por todos. El soporte de accesibilidad es necesario para permitir que la tecnología de asistencia interprete las páginas web.
```

File: content/docs/accessibility.md [hunk 2]
```diff
@@ -1,32 +1,32 @@
 ---
 id: accessibility
-title: Accessibility
+title: Accesibilidad
 permalink: docs/accessibility.html
 ---
 
-## Why Accessibility? {#why-accessibility}
+## ¿Por qué Accesibilidad?
```
Comment (alejandronanez) [inline content/docs/accessibility.md]: Bring those extra # back! :)
```suggestion
## ¿Por qué Accesibilidad? {#why-accessibility}
```

File: content/docs/accessibility.md [hunk 3]
```diff
@@ -1,32 +1,32 @@
...
+- [Lista de verificación WCAG de WebAIM](http://webaim.org/standards/wcag/checklist)
+- [Lista de verificación de El Proyecto A11Y](http://a11yproject.com/checklist.html)
 
 ### WAI-ARIA {#wai-aria}
 
-The [Web Accessibility Initiative - Accessible Rich Internet Applications](https://www.w3.org/WAI/intro/aria) document contains techniques for building fully accessible JavaScript widgets.
+El documento [Iniciativa de Accesibilidad Web - Aplicaciones de Internet Enriquecidas y Accesibles (WAI-ARIA por sus siglas en inglés)](https://www.w3.org/WAI/intro/aria) contiene técnicas para construir widgets de JavaScript totalmente accesibles.
 
-Note that all `aria-*` HTML attributes are fully supported in JSX. Whereas most DOM properties and attributes in React are camelCased, these attributes should be hyphen-cased (also known as kebab-case, lisp-case, etc) as they are in plain HTML:
+Tenga en cuenta que todos los atributos HTML `aria- *` son totalmente compatibles con JSX. Mientras que la mayoría de las propiedades y atributos de DOM en React son camelCase, estos atributos deben tener un guión (también conocido como kebab-case, lisp-case, etc.) ya que están en HTML simple:
```
Comment (alejandronanez) [inline content/docs/accessibility.md]:
```suggestion
Ten en cuenta que todos los atributos HTML `aria- *` son totalmente compatibles con JSX. Mientras que la mayoría de las propiedades y atributos de DOM en React son camelCase, estos atributos deben tener un guión (también conocido como kebab-case, lisp-case, etc.) ya que están en HTML simple:
```

Comment (ElRodrigote) [reply content/docs/accessibility.md]: Done!

File: content/docs/accessibility.md (hunk 1 above)
Comment (ElRodrigote) [reply content/docs/accessibility.md]: Done!

File: content/docs/accessibility.md (hunk 2 above)
Comment (ElRodrigote) [reply content/docs/accessibility.md]: Done!

**Comment by ElRodrigote** [comment]
Hi @alejandronanez !

Thanks for the corrections and the review. I've noticed a few orthographic errors too so I fixed them. Please, let me know if any further changes are needed.


Cheers!
Rodrigo Ali.

**Comment by ElRodrigote** [comment]
Thank you guys!

**Pull request merged**